src/Paket.preview3/Paket.fsproj

```diff
@@ -1,7 +1,7 @@
-<Project Sdk="FSharp.NET.Sdk;Microsoft.NET.Sdk">
+<Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
     <OutputType>Exe</OutputType>
-    <TargetFramework>netcoreapp1.0</TargetFramework>
+    <TargetFramework>netcoreapp2.0</TargetFramework>
```

Comment:
I don't think there is any downside to targeting netcoreapp2 instead of netcoreapp1, what do you think?

If I remember correctly, I had some kind of issue, but this net core stuff was a bit of trial and error, if you want I can try to downgrade again.